helm/charts/oathkeeper/README.md

```diff
@@ -32,6 +32,8 @@ A Helm chart for deploying ORY Oathkeeper in Kubernetes
 | deployment.annotations | object | `{}` |  |
 | deployment.automountServiceAccountToken | bool | `false` |  |
 | deployment.extraEnv | list | `[]` |  |
+| deployment.extraVolumeMounts | list | `[]` | Extra volume mounts, allows mounting the extraVolumes to the container. |
```

Comment:
This is autogenerated on CI, did you add it manually or regenerated with `helm-docs`?

Reply:
I regenerated with helm-docs